lib/sampler.js

```diff
@@ -15,12 +15,78 @@
  * limitations under the License.
  */
 
-class AlwaysSampledSampler {
-    isSampled() {
-        return true;
+const BIT_SET = Symbol('BIT_SET');
+const COUNTER = Symbol('COUNTER');
+
+function neverSample(trace) { // eslint-disable-line no-unused-vars
+    return false;
+}
+
+function alwaysSample(trace) { // eslint-disable-line no-unused-vars
+    return true;
+}
+
+class Sampler {
+    isSampled(trace) {} // eslint-disable-line no-unused-vars
+}
+
+class AlwaysSample extends Sampler {
+    isSampled(trace) {
+        return alwaysSample(trace);
+    }
+}
+
+class NeverSample extends Sampler {
+    isSampled(trace) {
+        return neverSample(trace);
```

Comment:
Here I'd prefer assigning function to variable:
```Javascript
class Sampler {
  isSampled(trace) {} // eslint-disable-line no-unused-vars
}

class AlwaysSample extends Sampler {
  constructor() {
    super();
    this.isSampled = alwaysSample;
  }
}

class NeverSample extends Sampler {
  constructor() {
    super();
    this.isSempled = neverSample;
  }
}
```
The benefit of this is you don't have to know what arguments will be used in `isSampled` function and in `neverSample`, `alwaysSample`. But on the other hand, it looks more ugly than returning result of function. So it's up to you what you will choose.